.devcontainer/devcontainer.json

```diff
@@ -8,6 +8,21 @@
 		"../compose.yml",
 		"compose.yml"
 	],
+	"features": {
+		"ghcr.io/devcontainers/features/common-utils:2": {
+			"installZsh": "true",
+			"username": "vscode",
+			"userUid": "1000",
+			"userGid": "1000",
+			"upgradePackages": "true"
+		},
+		"ghcr.io/devcontainers/features/github-cli:1": {},
```

Comment:
I personally don't use GitHub CLI, but not sure about others.

Reply:
Some people use git cli to create branches, push and pull.
I do . It will be useful if some issue happened and have to reset or force push etc.